classes/icon_helper.php

```diff
@@ -138,4 +138,15 @@ public static function space() : string {
             return $OUTPUT->pix_icon('i/empty', '');
         }
     }
+
+    /**
+     * Returns a permission icon.
+     *
+     * @return mixed An icon object for rendering.
+     */
+    public static function permission(string $str) {
```

Comment:
This is tripping a failure in the github actions pipeline:

blocks/multiblock/classes/icon_helper.php
    Line 142: Phpdocs for function icon_helper::permission has incomplete parameters list

if you could please update the PR to fix it so I can merge that would be great - thanks!!